src/encoded/auth0.py

```diff
@@ -96,6 +92,47 @@ def forget(self, request):
         return []
 
 
+def _signup(request):
+    """
+    Create new user.
+
+    :param request: Pyramid request object
+    """
+    domain = 'encode.auth0.com'
+    access_token = request.json['accessToken']
```

Comment:
This still seems problematic if the request doesn't have accessToken.

Reply:
Ok, I will do a check and blow up if accessToken is not found. But it also my be a good to leave it because if accessToken is not available, the user has an invalid request or is trying to hack the system; so the code should blow up.

Reply:
I now check for accessToken and raise an exception if not available